curve_apps/edge_detection/params.py

```diff
@@ -37,9 +37,9 @@ class SourceParameters(BaseModel):
     data: FloatData
 
 
-class DetectionParameters(BaseModel):
+class EdgeDetectionParameters(BaseModel):
```

Comment:
Do we need the `Edge` since under the `edge_detection` module?

Reply:
After writing tests and running into name clashes, I think we should be explicit about the names..